ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/activity/TutorialActivity.kt

```diff
@@ -21,7 +21,7 @@ class TutorialActivity : AppCompatActivity(), TutorialCallbacks {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_baked_in_onboarding)
+        setContentView(R.layout.activity_tutorial)
         viewPager = findViewById(R.id.baked_in_viewpager)
```

Comment:
Is this view pager specific to the baked in state, or is it general for both states?

Reply:
for both

Reply:
If that's the case, we should probably rename it to reflect that fact.